### PR TITLE
Notes page

### PR DESCRIPTION
**Pull request by sanathkumar26700**
Created notes page which contains,
 
- Notes section,
     1. Note modal with a rich text editor
     2. On creating a note, it will be displayed in the notes page
     3. Notes can pinned, unpinned and edited 

- Filter section where notes can be filtered by,
     1. Tags
     2. Priorities
     3. Time of creation
     4. Search bar
     5. Filters can be reset

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
Good to go.

File: src/Components/Dark mode/darkMode.jsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```
Comment (msdeshmukh009) [inline src/Components/Dark mode/darkMode.jsx:1]: You don't need this import.

**Review by manojasarsa — APPROVED** [review]
The code looks clean and structured.
GTG!

**Review by msdeshmukh009 — APPROVED** [review]
All the mentioned features are working. Good to go.

File: src/Components/Dark mode/darkMode.jsx (hunk 1 above)
Comment (sanathkumar26700) [reply src/Components/Dark mode/darkMode.jsx:1]: Yes, I'll remove it

**Pull request merged**